spec/models/form_profile_spec.rb

```diff
@@ -526,7 +529,7 @@
           'productId' => '1',
           'availableForReorder' => false,
           'lastOrderDate' => '2020-01-01',
-          'nextAvailabilityDate' => '2020-09-01',
+          'nextAvailabilityDate' => '2099-09-01',
```

Comment:
ah haha no reason other than I saw that it would be considered availableForReorder (which is currently false) on September 1st, 2020 so I just quickly changed it to something far in the future so I wouldn't have to worry about that.